core/components/atoms/_overlay/overlay.js

```diff
@@ -76,22 +91,28 @@ Overlay.Backdrop = styled.div`
   bottom: 0;
   right: 0;
   z-index: ${layers.overlayBackdrop};
-  background: rgba(0, 0, 0, 0.7);
+  background: hsla(0,12%,95%,.95);
```

Comment:
rgba -> hsla? 

I'm a color noob, should we prefer one over the other?

Reply:
I don't think it's a big deal, this is to match exactly the color we have today at style guides.

We'll standarize once we create the variable system, hopefully soon enough